README.md

```diff
@@ -21,6 +21,14 @@ along with this program. If not, see <https://www.gnu.org/licenses/>.
 # Blue Brain Search
 
 <table>
+<tr>
+  <td>DOI</td>
```

Comment:
```suggestion
  <td>Source Code DOI</td>
```
We'll have another entry in this table for the `Data DOI`, so what about being more specific here and say `Source Code DOI` or `Software DOI` or something like that?